### PR TITLE
Add Mint Fees as config param to claims

### PR DESCRIPTION
**Pull request by richerd**
## This PR updates mint fees to be initialization params 

moves MINT_FEE and MINT_FEE_MERKLE to be constructor params to allow for variable mint fee to for allowing for different fees to be charged on different networks.

### CODE REVIEW

File: packages/manifold/contracts/lazyclaim/ERC1155LazyPayableClaim.sol [hunk 1]
```diff
@@ -33,8 +33,7 @@ contract ERC1155LazyPayableClaim is IERC165, IERC1155LazyPayableClaim, ICreatorE
             interfaceId == type(IERC165).interfaceId;
     }
 
-    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2) LazyPayableClaim(initialOwner, delegationRegistry, delegationRegistryV2) {}
-
+    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2, uint256 mintFee, uint256 mintFeeMerkle) LazyPayableClaim(initialOwner, delegationRegistry, delegationRegistryV2, mintFee, mintFeeMerkle) {}
```
Comment (wwhchung) [inline packages/manifold/contracts/lazyclaim/ERC1155LazyPayableClaim.sol:36]: Fix ordering, put mint fees first.

File: packages/manifold/contracts/lazyclaim/ERC721LazyPayableClaim.sol [hunk 2]
```diff
@@ -47,7 +47,7 @@ contract ERC721LazyPayableClaim is IERC165, IERC721LazyPayableClaim, ICreatorExt
             interfaceId == type(IERC165).interfaceId;
     }
 
-    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2) LazyPayableClaim(initialOwner, delegationRegistry, delegationRegistryV2) {}
+    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2, uint256 mintFee, uint256 mintFeeMerkle) LazyPayableClaim(initialOwner, delegationRegistry, delegationRegistryV2, mintFee, mintFeeMerkle) {}
```
Comment (wwhchung) [inline packages/manifold/contracts/lazyclaim/ERC721LazyPayableClaim.sol:50]: Fix ordering, put mint fees first (convention is to put static variables first.

File: packages/manifold/contracts/lazyclaim/LazyPayableClaim.sol [hunk 3]
```diff
@@ -72,12 +72,13 @@ abstract contract LazyPayableClaim is ILazyPayableClaim, AdminControl {
         _;
     }
 
-    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2) {
+    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2, uint256 mintFee, uint256 mintFeeMerkle) {
```
Comment (wwhchung) [inline packages/manifold/contracts/lazyclaim/LazyPayableClaim.sol:75]: Fix ordering, put mint fees first (convention is to put static variables first.

**Comment by wwhchung** [comment]
You need to update the deploy scripts and set it to switch on a network environment variable.  The deploy scripts should have the hardcoded amounts.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -20,10 +20,10 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
 
-      - name: Set Node to v16
+      - name: Set Node to v18
```
Comment (campionfellin) [inline .github/workflows/ci.yml:23]: I think we're on 20 now

**Comment by wwhchung** [comment]
Closing this because it won't work and will result in different contract addresses across different networks.

**Pull request closed without merging**